### ISSUE TITLE
Must a site/PWA have a manifest with `shortcuts` to use this API?

### ISSUE THREAD

**Issue by aarongustafson** [rayankans/app-shortcuts#6]
My feeling is "no," which could open the door to allowing sites to define Shortcuts for their tab.

**Comment by beaufortfrancois** [comment]
If that wasn't the case, I believe we shouldn't have used the web app manifest and instead simply started with this JS API.

**Comment by rayankans** [comment]
Agreed, I think the website should at the very least have a manifest defined. Shortcuts make sense within the context of an "app".

**Comment by aarongustafson** [comment]
> Shortcuts make sense within the context of an "app".

I agree. I do think it’s worth considering that features like this did exist for Pinned Sites (which were not "installed" in the traditional sense) and we’ve been seeing some usage of "Install this site as an app" in Edge, which blurs the line too. I’ve also heard some developers say this would be really interesting for Pinned Tabs and tabs in general, at least on desktop.

**Comment by rayankans** [comment]
I wouldn't say it's gated as much as having a valid manifest is a prerequisite for the API to work as expected. This is a bit like `getInstalledRealtedApps` which  still works without a manifest but doesn't do anything useful unless there is a manifest with related_applications defined.
